scipy/signal/lti_conversion.py

```diff
@@ -359,6 +359,7 @@ def cont2discrete(system, dt, method="zoh", alpha=None):
             * euler: Euler (or forward differencing) method ("gbt" with alpha=0)
             * backward_diff: Backwards differencing ("gbt" with alpha=1.0)
             * zoh: zero-order hold (default)
+            * foh: first-order hold
```

Comment:
we sometimes use `versionadded` directives for feature additions, though it can be awkward in some parts of the docstring; maybe wait for feedback from signal processing experts first

Reply:
yes, this would be good. the Sphinx directive won't work here, so I suggest to change this line like so:
```
foh: first-order hold (*versionadded: 1.3.0*)
```
and the same for `impulse`

Reply:
OK, just a minute.